### PR TITLE
feat: added homepage

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
Added the homepage layout for e-comm Style Store
image

image

### CODE REVIEW

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.main-container{
+    margin: 9vh 1rem 0 1rem;
+    display: grid;
+    grid-template-rows: 1fr 3fr 2fr;
+    row-gap: 1rem;
+}
+
+/* .main-container>*{
```
Comment (sameep-s) [inline css/homepage.css]: you can remove these unused comments.

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: coral;
+} */
+
+.categories{
+    height:fit-content;
+    width: 100%;
+    display:flex ;
+    flex-direction: row;
```
Comment (sameep-s) [inline css/homepage.css]: flex direction is by default row so you can remove it if it's not useful.

File: css/homepage.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply css/homepage.css]: sure

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,80 @@
+:root{
+    --primary-color:black;
+    --logo-color:#f2f2f2;
+    --white-color:#fff;
+    --icon-badge:#0000ee;
+}
+
+*{
+    /* border: 1px solid white; */
```
Comment (sameep-s) [inline styles.css]: unused css you can remove it.

File: css/homepage.css (hunk 2 above)
Comment (muskan-gupta-0803) [reply css/homepage.css]: okay, thanks for reviewing.

**Review by sameep-s — APPROVED** [review]
Just look at the comments , after that you are good to go.

**Comment by muskan-gupta-0803** [comment]
> Just look at the comments , after that you are good to go.

sure, changes done. thanks

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Looks good.
just Add deploy link

**Comment by muskan-gupta-0803** [comment]
> > Just look at the comments , after that you are good to go.
> 
> sure, changes done. thanks

Sure will add, thanks for review

**Comment by muskan-gupta-0803** [comment]
> Looks good. just Add deploy link

Sure, will add . thanks for review

**Pull request merged**